policies/verify-tenant-patch/policies/rule_rego_test.rego

```diff
@@ -0,0 +1,556 @@
+package crownlabs_tenant_patch
+
+#creator doesn't have any tenant
+test_no_tenant_present {
+	input := {"review": input_review7}
```

Comment:
Would it be possible to use more explicit names to explain the content of the input_review, to make them easier to understand? e.g. input_review1 -> input_review_creator_has_no_tenant